Library/Homebrew/cleanup.rb

```diff
@@ -350,10 +359,12 @@ def cache_files
       files = cache.directory? ? cache.children : []
       cask_files = (cache/"Cask").directory? ? (cache/"Cask").children : []
       api_source_files = (cache/"api-source").glob("*/*/*/*/*") # org/repo/git_head/type/file.rb
+      gh_actions_artifacts = (cache/"gh-actions-artifact").directory? ? (cache/"gh-actions-artifact").children : []
```

Comment:
I think making this an `if` or `||=` would be easier to follow.

Reply:
Personally I think that does read a bit nicer. Could always have a

```ruby
else
  []
end
``` 

if you wanted to avoid `to_a`. Fine with me to leave as-is though, don't feel strongly.

Reply:
Passing on this for now, but won't object to someone refactoring this bit later on.